lib/eco/eco.py

```diff
@@ -466,6 +450,8 @@ def change(self, index):
             self.ui.textEdit.setText(text)
         elif index == "Default":
             text = self.tm.export("/dev/null")
+            if text is None:
+                text = ""
```

Comment:
These were two lines that I added, because I saw a traceback when I was testing these changes. Perhaps this should have been a separate PR?